charts/lagoon-core/templates/k8up.schedule.yaml

```
@@ -0,0 +1,24 @@
+{{ if .Capabilities.APIVersions.Has "backup.appuio.ch/v1alpha1/Schedule" }}
+apiVersion: backup.appuio.ch/v1alpha1
+kind: Schedule
+metadata:
+  name: 'lagoon-core-schedule'
```

could we change this to not be hard-coded? that way it is consistent with the other templates.
```suggestion
  name: {{ include "lagoon-core.fullname" . }}
```